packages/cozy-pouch-link/src/PouchManager.js

```diff
@@ -97,66 +97,33 @@ export default class PouchManager {
     // We must ensure databases exist on the remote before
     // starting replications
     this.ensureDatabasesExistDone = false
+
+    this.startReplicationLoop = this.startReplicationLoop.bind(this)
+    this.stopReplicationLoop = this.stopReplicationLoop.bind(this)
   }
 
   addListener() {
-    if (isMobileApp() && !this.listenerLaunched) {
-      document.addEventListener(
-        'pause',
-        () => this.stopReplicationLoop(),
-        false
-      )
-      document.addEventListener(
-        'resign',
-        () => this.stopReplicationLoop(),
-        false
-      )
-      document.addEventListener(
-        'resume',
-        () => this.startReplicationLoop(),
-        false
-      )
-      document.addEventListener(
-        'offline',
-        () => this.stopReplicationLoop(),
-        false
-      )
-      document.addEventListener(
-        'online',
-        () => this.startReplicationLoop(),
-        false
-      )
+    if (!this.listenerLaunched) {
+      if (isMobileApp()) {
+        document.addEventListener('pause', this.stopReplicationLoop, false)
+        document.addEventListener('resign', this.stopReplicationLoop, false)
+        document.addEventListener('resume', this.startReplicationLoop, false)
+      }
+      document.addEventListener('online', this.startReplicationLoop, false)
+      document.addEventListener('offline', this.stopReplicationLoop, false)
```

Comment:
`false` is the default value for the `useCapture` parameter of `addEventListener`. Did you see something that forced you to specify it?